include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -296,9 +296,8 @@ SECTIONS
 
 	_app_smem_size = _app_smem_end - _app_smem_start;
 	_app_smem_rom_start = LOADADDR(_APP_SMEM_SECTION_NAME);
-#endif  /* CONFIG_USERSPACE */
 
-    SECTION_DATA_PROLOGUE(_BSS_SECTION_NAME,(NOLOAD),)
+   SECTION_DATA_PROLOGUE(_BSS_SECTION_NAME,(NOLOAD),)
```

Comment:
This whitespace change should not be present; that is, unless you are willing to clean up the entire file while you are at it.